Kitodo/src/test/java/org/kitodo/selenium/testframework/pages/ProjectEditPage.java

```diff
@@ -43,14 +43,6 @@
     @FindBy(className = "ui-selectonemenu-trigger")
     private WebElement selectTrigger;
 
-    @SuppressWarnings("unused")
```

Comment:
Why did you remove this and other lines in this file? This does not look right for me as you did not change anything related to project editing.

Reply:
I removed this lines because there is no need to add explicitly the startDateInput and the endDateInput for tests and instead of that, I used implicitly the default input values of this dates (today as default value) which are already entered automatically in the form  and this is why I didn't change any other thing.
Deleting this lines and using the default values of dates eliminate the problem of  date format when the UI is in English(08/05/18) or in German("05.08.19").

Reply:
I'm not sure why this was added by @Beacze. I think there is a reason for it. Maybe @Beacze can answer if this could really be removed or solved in an other way?

Reply:
I have discussed this with @IkramMaalej. I need to check what exactly causes test fails after she adds possiblity for user to manually change the language. Until then this part of code can be commented out / removed.

Reply:
Thank you @Beacze.